cardano-api/src/Cardano/Api.hs

```diff
@@ -382,6 +417,10 @@ buildShelleyTransaction txins txouts ttl fee certs = do
    certDiscrim (ShelleyGenesisDelegationCertificate sGenDelegCert) = sGenDelegCert
    certDiscrim (ShelleyMIRCertificate mirCert) = mirCert
 
+   updateDiscrim :: Maybe Update ->  Shelley.StrictMaybe ShelleyUpdate
```

Comment:
Slightly odd name for that. `toStrictMaybe` instead?

In fact, isn't that a lib function somewhere?

Reply:
`maybeToStrictMaybe :: Maybe a -> StrictMaybe a` lives in the ledger specs library but my function is `a -> b`. I can rename it.